test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -153,10 +153,7 @@ public Task DefaultsServerAddress_BindsToIPv6()
 
             var hostBuilder = new WebHostBuilder()
                .UseKestrel()
-               .ConfigureServices(services =>
-               {
-                   services.AddSingleton<ILoggerFactory>(new KestrelTestLoggerFactory(testLogger));
-               })
+               .UseLoggerFactory(_ => new KestrelTestLoggerFactory(testLogger))
```

Comment:
We were thinking of obsoleting that overload. https://github.com/aspnet/Hosting/issues/1007. We want to use the overloads that also take in the WebHostBuilderContext.